src/lib/error-handling.js

```diff
@@ -13,10 +13,11 @@ module.exports = {
     let redirect_uri = req.session.authParams.redirect_uri;
 
     if (err.isAxiosError) {
-      const oauthError = err?.response?.data?.oauth_error;
+      const oauthError = err?.response?.data;
```

Comment:
Have you run this locally on the Fraud / KBV repos using yarn link? 

I'd expect that changing the properties of what is considered an error would break the tests

Reply:
e.g. https://github.com/alphagov/di-ipv-cri-kbv-front/blob/main/test/mocks/mappings/answer-error.json